web/admin/panels/Activities.js

```diff
@@ -339,6 +340,11 @@ function ActivitiesPanel() {
             }}
             cancelText={null}
             maxDate={today}
+            // renderDay is needed to fix issue caused by key being passed with props
```

Comment:
ça supprime bien l'erreur "key is passed to jsx, ..." mais j'ai l'impression que ça a fait apparaître une erreur 

Warning: Each child in a list should have a unique "key" prop.
Check the render method of `DayPicker`.